modules/administration-guide/examples/prometheus-config.adoc

```diff
@@ -14,4 +14,7 @@ data:
           static_configs:
             - targets: ['[che-host]:8087']  <4>          
 ----
-
+<1> The rate at which a target is scraped.
```

Comment:
Just to confirm that this yaml block is also the same for the CRW.

Reply:
Yes, same for CRW

Reply:
Thank you. Feel free to resolve this thread.